baremetal_vm/create_vm.sh

```diff
@@ -136,7 +136,8 @@ spec:
     checksum: "http://$IP_ADDR:8080/${IMG_FILENAME}.md5sum"
   bootMACAddress: $NODEMAC
   bootMode: legacy
-  
+  rootDeviceHints:
+    deviceName: ${DEVICE_HINT}
```

Comment:
We should default this e.g [here](https://github.com/suse-edge/misc/blob/main/baremetal_vm/common.sh#L28) otherwise the variable will be undefined when it's not explicitly set in the env file

Reply:
done! you're right